feature-account-impl/src/main/java/io/novafoundation/nova/feature_account_impl/domain/cloudBackup/enterPassword/EnterCloudBackupInteractor.kt

```diff
@@ -26,30 +23,19 @@ interface EnterCloudBackupInteractor {
 class RealEnterCloudBackupInteractor(
     private val cloudBackupService: CloudBackupService,
     private val cloudBackupFacade: LocalAccountsCloudBackupFacade,
-    private val accountRepository: AccountRepository,
+    private val accountRepository: AccountRepository
 ) : EnterCloudBackupInteractor {
 
     override suspend fun restoreCloudBackup(password: String): Result<Unit> {
         return cloudBackupService.fetchAndDecryptExistingBackup(password)
             .mapCatching { cloudBackup ->
                 // `CannotApplyNonDestructiveDiff` shouldn't actually happen here since it is a import for clean app but we should handle it anyway
-                val diff = cloudBackupFacade.applyNonDestructiveCloudVersionOrThrow(cloudBackup, BackupDiffStrategy.importFromCloud())
+                cloudBackupFacade.applyNonDestructiveCloudVersionOrThrow(cloudBackup, BackupDiffStrategy.importFromCloud())
 
                 val firstSelectedMetaAccount = accountRepository.getActiveMetaAccounts().first()
                 accountRepository.selectMetaAccount(firstSelectedMetaAccount.id)
 
                 cloudBackupService.session.initEnabledBackup(password)
-
-                diff
-            }.flatMap { diff ->
-                // Once we successfully applied state locally, we can write new changes to backup to sync backup with other local state
-                if (diff.cloudChanges.isEmpty()) return Result.success(Unit)
-
-                val request = WriteBackupRequest(
-                    cloudBackup = cloudBackupFacade.fullBackupInfoFromLocalSnapshot(),
-                    password = password
-                )
-                cloudBackupService.writeBackupToCloud(request)
```

Comment:
I thought that we should trigger cloud backup sync in this case but I forgot that in this case it wouldn't be triggered since we don't handle backup sync from this source.

Thank you for your note